Bottom: e8ec89e364cf7a82f6fee493800af00ccb8805da
Top:    d75cd2f24aae22b576d750c98dff4e06c5e21025
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-14 13:19:57 +0100

Avoid clearing pins / vias from polygons... it is already done!




---

diff --git a/src/draw.c b/src/draw.c
index 9c6d8ee..d52515f 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1238,6 +1238,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1260,6 +1261,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
